erpnext/hr/doctype/employee/employee.json

```diff
@@ -2234,7 +2263,7 @@
  ], 
  "hide_heading": 0, 
  "hide_toolbar": 0, 
- "icon": "fa fa-user", 
+ "icon": "icon-user", 
```

Comment:
This is some issue, every time you change something in doctype, it changes back to icon-user.